### ISSUE TITLE
 Accessibility: Change code for B logo in menu to accommodate Screen Readers 

### ISSUE THREAD

**Issue by BrutiAnteneh** [CityOfBoston/patterns#288]
### The Problem: 
Siteimprove flags the B in the Menu as problematic for assistive technology because a **title tag** is present without an accompanying **alt tag**.  A screen reader knows how to handle both an empty alt attribute and one with a text. If there is no attribute some screen readers will compensate and read the path to the image instead.

![site improve ticket screen shot](https://user-images.githubusercontent.com/33470951/37884745-79718c3a-307f-11e8-8244-e2c0c0f058cc.png)

![b logo site improve alt text screen shot](https://user-images.githubusercontent.com/33470951/37884480-1ffac208-307e-11e8-8ef8-8dd9edf0d08b.png)

### View Issue and Suggested Solutions in SiteImprove
URL: https://my2.siteimprove.com/Inspector/341026/Accessibility/Page?pageId=8050765329&impmd=68716A68AC2C7A88B4D3675E121C85F5#/Criterion/1.1.1/Check/1

Summary of suggested solutions:
1. add **alt text** to B image in the Menu to resolve the issue for screen readers, or
2. remove **title** attribute and set the alt text to null (**alt=“”**), if the preference is to have screen readers _ignore_ the logo.

### Fixing this addresses other Accessibility issues flagged in SiteImprove:

**The image does not have the correct alternative:** 
URL: https://my2.siteimprove.com/Inspector/341026/Accessibility/Page?pageId=8050765329&impmd=68716A68AC2C7A88B4D3675E121C85F5#/Criterion/1.1.1/Check/46

**Image link is missing alternative text**
URL: https://my2.siteimprove.com/Accessibility/341026/Pages/Issue?CheckId=5&CriterionChapter=2.4.4

**Comment by reillyzlab** [comment]
Happening on both sites, so make sure resolved on both before passing.

**Comment by fionawhim** [comment]
`alt` should just be `""` to hide it.

**Comment by reillyzlab** [comment]
@davidrkupton or @kdonaldson can either of you take this?

**Comment by kdonaldson** [comment]
@reillyzlab what would you like the alt text to be?

**Comment by duffy-james** [comment]
Let's go with number 2:

- Remove title attribute and set the alt text to null (alt=“”)

@kdonaldson

**Comment by BrutiAnteneh** [comment]
The issue passed as expected.
